AirCasting/Settings/BackendURLValidator.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  BackendURLValidator.swift
+//  URLValidation
+//
+// Created by Lunar on 25/06/2021.
+//
+
+import Foundation
+
+class BackendURLBuilder {
```

Comment:
I missed this one last time :( can you change the name of this file to match the class name?